src/solver.cpp

```diff
@@ -113,6 +113,7 @@ struct IdRegistryEntry
     Primitive primitive            = Primitive::Convolution;
     miopenConvAlgorithm_t convAlgo = miopenConvolutionAlgoDirect;
     AnySolver solver               = {};
+    bool is_tunable                = false;
```

Comment:
I think this should be a Solver's attribute, not SolverID's one. We can easily implement a generic function that takes a Solver instance and returns if it is tunable or not, and a wrapper in `AnySolver`.

@DrizztDoUrden Am I correct?

Reply:
I think it makes sense as an AnySolver attribute. @JehandadKhan suggested doing it this way would be a smaller diff, and wouldn't require editing all of the solvers.

Reply:
I think it should be a solver attribute. Basically, it describes solver implementation details.

Reply:
@cderb Since @DrizztDoUrden and @atamazov both agree that this should be a solver attribute, can you please update your PR?

Reply:
I'm testing a solution.